### PR TITLE
Add basic stress test txns

### PR DESCRIPTION
**Pull request by zivkovicmilos**
# Description

This PR just adds 2 simple tests that stress test the consensus module.

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have tested this code
- [ ] I have updated the README and other relevant documents (guides...)
- [x] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

File: e2e/StressTest.sol [hunk 1]
```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.8.0;
+
+import "hardhat/console.sol";
```
Comment (Kourin1996) [inline e2e/StressTest.sol]: It should work without this import

File: e2e/transaction_test.go [hunk 2]
```diff
@@ -265,3 +274,287 @@ func TestEthTransfer(t *testing.T) {
...
+	}
+
+	return bigResponse, nil
+}
+
+// Test scenario (Dev mode):
+// Deploy the StressTest smart contract and send ~50 transactions
+// that modify it's state, and make sure that all
+// transactions were correctly executed
+func Test_TransactionDevLoop(t *testing.T) {
```
Comment (Kourin1996) [inline e2e/transaction_test.go]: Is it possible to share a part of codes with `Test_TransactionIBFTLoop`?

**Review by Kourin1996 — CHANGES_REQUESTED** [review]

File: e2e/StressTest.sol (hunk 1 above)
Comment (zivkovicmilos) [reply e2e/StressTest.sol]: Got it, thank you for pointing it out 👍 

65759db

File: e2e/transaction_test.go (hunk 2 above)
Comment (zivkovicmilos) [reply e2e/transaction_test.go]: Yep, fixed 👍 

4e257ee

**Review by Kourin1996 — APPROVED** [review]
LGTM 👍

**Review by lazartravica — APPROVED** [review]
👌  Please resolve the merge conflicts and feel free to merge

**Pull request merged**